Core/clim-core/frames.lisp

```diff
@@ -725,6 +728,29 @@ documentation produced by presentations.")
 ;
 ; FIXME
 
+(defun update-frame-pane-lists (frame)
+  (let ((all-panes     (frame-panes frame))
+        (named-panes   (mapcar #'cdr (frame-panes-for-layout frame)))
+        (current-panes '()))
+    ;; Find intersection of named panes and current layout panes.
+    (map-over-sheets (lambda (sheet)
+                       (when-let ((index (position sheet named-panes)))
+                         (push (cons sheet index) current-panes)))
+                     all-panes)
+    (setf current-panes (map 'list #'car (sort current-panes #'< :key #'cdr)))
+    ;; Populate current-pane list and special pane slots.
+    (let ((interactor (find-pane-of-type current-panes 'interactor-pane)))
```

Comment:
is there a reason for preemptively find interactor pane, but not application and pointer-documentation panes? if not, please add (application (find-pane-of-type … 'application-pane)) and (pointer-doc …) and use them accordingly.

Reply:
The `interactor` variable is used in multiple places which is not the case for the results of the other `find-pane-of-type` results

Reply:
We look for each pane anyway, so there is no harm of putting them in LET and it (matter-of-taste-speaking) makes the code more readable.

Reply:
Right, will do.

Reply:
Changed.